### ISSUE TITLE
midi instruments should all play the same midi riff

### ISSUE THREAD

**Issue by RustoMCSpit** [studiorack/studiorack-site#31]
there should be a single lead melody which all the instruments play

**Comment by kmturley** [comment]
Yes good idea, I thought of something similar. There are some issues with implementing this:

1. Different instruments require different articulations e.g. Violin vs Piano vs Synth which would not show off the instrument to it's full potential
2. Creating the samples would be alot of work. I was hoping for a GitHub workflow solution, but there are some limitations.
Sampled instruments in sfz could be played with something like https://github.com/sfztools/sfizz-render but other instruments would need a to run headless and capture the audio somehow.

Or would have to do it manually. Maybe a middle-ground solution would be to provide a template authors can use as a starting point.

For now I created all previews using a middle-C note, but that does not really show off the low instruments vs high.

Ideal long term solution is to actually let the user play the isntrument in the browser.
I made some progress in this space, and with web modules it may be possible. But this would be a long-term goal and I have plenty of other features to fix/build before that!

**Comment by RustoMCSpit** [comment]
https://github.com/orgs/studiorack/projects/3/views/2?pane=issue&itemId=7463818
https://github.com/orgs/studiorack/projects/3/views/2?pane=issue&itemId=19156666
